src/Evolution/Systems/GrMhd/ValenciaDivClean/Characteristics.hpp

```diff
@@ -112,6 +112,7 @@ std::array<DataVector, 9> characteristic_speeds(
     const EquationsOfState::EquationOfState<true, ThermodynamicDim>&
         equation_of_state) noexcept;
 
+namespace Tags {
```

Comment:
while you are making this change, can you rename the `ComputeTag` struct to `CharacteristicSpeedsCompute` as @nilsdeppe pointed out that such a naming convention has the advantage that in the documentation the compute tags will show up next to their simple tags